sale_margin_percentage/models/sale_order.py

```diff
@@ -33,17 +32,19 @@ class SaleOrderLine(models.Model):
     _inherit = "sale.order.line"
 
     @api.model
-    def _default_margin_threshold(self):
+    def _get_default_margin_threshold(self):
```

Comment:
@deivislaya 

please, leave this method as `_default_margin_threshold`..

by [convention](https://www.odoo.com/documentation/9.0/reference/guidelines.html#symbols-and-conventions) is the right name

regards,

Reply:
Thanks for the info, change applied

Reply:
+1